docs/source/miniconda.rst

```diff
@@ -14,6 +14,21 @@ packages from the Anaconda repository.
 
 `See if Miniconda is right for you <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html#anaconda-or-miniconda>`_.
 
+
+System requirements
+===================
+
+* License: Free use and redistribution under the terms of the `EULA for Anaconda Individual Edition <https://www.anaconda.com/end-user-license-agreement-miniconda>`_. 
```

Comment:
The Link text for this should read EULA for Miniconda not Anaconda Individual Edition